src/ReservoirOracleUnderwriter.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.17;
+pragma solidity >=0.8.0;
```

Comment:
shouldn't we use same version for all files?

Reply:
I think most of these can have an open solc. PaprController is what will compile them all and will force all to what it uses